src/main.ts

```diff
@@ -999,43 +948,3 @@ const cleanUpProxies = async () => {
     kill(proc.pid)
   })
 }
-
-const createFileWithUniqueName = async ({
```

Comment:
This function is being used more than once in `main.ts` for different handlers, so I moved it to a utility file.